src/routes/safe/components/Transactions/TxsTable/columns.js

```diff
@@ -28,6 +29,14 @@ type TxData = {
 
 export const formatDate = (date: string): string => format(parseISO(date), 'MMM d, yyyy - HH:mm:ss')
 
+export const getIncomingTxAmount = (tx: IncomingTransaction) => {
+  const {
+    utils: { toBN, fromWei },
+  } = getWeb3()
+
+  return Number(tx.value) > 0 ? `${fromWei(toBN(tx.value), 'ether')} ${tx.symbol || '???'}` : 'n/a'
```

Comment:
does this work for ERC20 tokens with decimals other than 18?

Reply:
Indeed, fixed. Thanks!